WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -50,7 +52,8 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Attribute metadata:
         + `dateUpdated`: Timestamp when the last update of the attribute happened.
             + Type: [DateTime](http://schema.org/DateTime)
-        + `TimeInstant` : Timestamp saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
+        + `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```

Comment:
Just confirming with you that you really want to link it to the `develop` branch of the https://github.com/telefonicaid/iotagent-node-lib project instead of `master` (where the stable, not the development version is) :)